.github/workflows/pr-handling.yaml

```diff
@@ -0,0 +1,38 @@
+name: Assign PR to author and reviewers
+
+# This workflow runs on the pull_request_target event:
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+#
+# So the secrets are accessible even if the PR was opened from an external
+# repository. This is done because the GitHub API key needs to be accessed
+# to modify the PRs.
+#
+# NOTE: Only API calls should be made in the actions defined here. The
+#       committed code should _NOT_ be touched in any case.
+
+on:
+  pull_request_target:
```

Comment:
From https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
```
pull_request_target runs in the context of the target repository of the PR, rather than in the merge commit. This means the standard checkout action uses the target repository to prevent accidental usage of the user supplied code.
```

Therefore, my understanding is that even if a malicious user modifies `scripts/pull-review.sh` - the original `pull-review.sh` from the target repo will be executed. So that makes it safe for inclusion, to my understanding.